modules/experimental/pages/portal.adoc

```diff
@@ -0,0 +1,139 @@
+See <<Portal Reserves>> for more about this.
+
+TIP: We use `_(Roll)_` and `_(Auto)_` to represent different technologies for different factions.
+They are mutually exclusive, for both thematic and practical reasons -- so {portal} units in a given army list must all be of one of these types, never a mix of both.
+
+== Game effects of {portal}
+
+Where your army includes any units or objectives that have the {portal} ability:
+
+* Regardless of the scenario, you may keep any detachments that contain units that can use a portal out of the play area, as <<Portal Reserves>>.
+
+Where your unit or objective has the {portal} ability:
+
+* You can use this unit to move portal reserves detachments into play after they arrive.
+You can also move detachments that are in play between one portal and another.
+See <<portal-moves>>.
+
+[[portal-moves]]
+== Use a {portal} unit to move your forces
+
+You can use portals to move some of your forces around.
+Where you wish to move one of your detachments into or out of a portal, and it contains permitted units:
+
+* You may do this only in the Movement phase.
+// TODO: Is this stipulation necessary? On the one hand we're trying to apply standard transport rules for simplicity, but on the other hand this kind of stipulation sets up differences from those rules which complicates things again and creates logical gaps.
+* You may do this only if you give the detachment Advance or Assault orders.
+* Use the normal xref:main-rules:transportation.adoc[] rules to embark and disembark the detachment's units.
+footnote:[
+When you disembark units from an *objective* that has the {portal} ability, you obviously cannot pay 5 cm of its movement like you would with a mobile transport unit.
+In this case, just deduct 5 cm from the movement of each unit that you disembark, as usual, and ignore the deduction that would normally apply to the transport unit.
+]
+// TODO: Do we need to mention something to the effect that portal reserves detachments either (a) already count as embarked so don't pay 5 cm penalty to embark, or (b) need to pay the penalty to embark, just as units already in play need to do? Also, given that for a detachment that is already in play you must embark and then immediately disembark from another portal, won't this get a bit complicated given that you must also pay a 5 cm penalty for *each* portal unit...? 
+* You may use each individual portal only *once per turn*, to embark or disembark a single detachment.
+Any portal that you use becomes unavailable until the start of the next turn.
+* You may move a detachment that is already in the play area into an available portal -- but only if you immediately disembark the detachment from a different available portal elsewhere in the play area.
+You cannot use portals to transfer detachments *out* of the play area during the game.
```

Comment:
Could this be simplified? You're adding an extra clause to cover something that can't happen, by inferring that there is a choice. Maybe something this instead?

```suggestion
* You may embark a detachment that is already in the play area into an available portal -- but you *must* immediately disembark the detachment from a different available portal elsewhere in the play area.
```

Reply:
Hmm. I was trying to pre-empt the question and make it really clear that players can't use portals to take units out of the play area — especially given that transport aircraft **do** allow this. (Again, we can view this as a difference from standard transportation rules, or at least a difference in how they apply to portals as compared with transport aircraft.)

I think that your suggestion works, and it's good to get the word 'embark' in here — but I wonder if it still makes sense to include the distinct point I was trying to make somewhere (perhaps as a note or footnote instead)?

Reply:
If you think it's needed.

I don't think it is, because the revised suggestion only gives you a single option for what happens when a unit embarks into a Portal.

Hmm, that reminds me, it's units, not detachments. Revised suggestion:

```suggestion
* You may embark a unit that is already in the play area into an available portal -- but you *must* immediately disembark the unit from a different available portal elsewhere in the play area.
```

Reply:
> If you think it's needed.
> I don't think it is, because the revised suggestion only gives you a single option for what happens when a unit embarks into a Portal.

I take your point, in line with the principle that if the rules don't actually say you can do something then you should assume that you can't. But I lean towards making it explicit that we do indeed intend that there is no option, at any point, to use a portal to remove units from the play area, and that it's not just something that we neglected to mention at this particular point. The more so given that:

* This is in contrast to transport aircraft.
* This is in contrast to the NetEA Necron portal rules for Epic Armageddon, which may be more familiar to some. (Very much a low priority consideration, but still, another small point in favour of being explicit.)

---

> Hmm, that reminds me, it's units, not detachments. Revised suggestion:

It's tricky because in this topic we often need to speak of putting _detachments_ into play, being able to use a given portal only once per turn to embark or disembark the units of one _detachment_, and so on.

This one is perhaps arguable, but I'm happy to go with your suggestion here.

---

So, putting both bits together here:

```suggestion
You may embark a unit that is already in the play area into an available portal -- but you *must* immediately disembark the unit from a different available portal elsewhere in the play area.
footnote:[
To be clear, there is no option to use a portal to transfer units *out* of the play area during the game.
]
```